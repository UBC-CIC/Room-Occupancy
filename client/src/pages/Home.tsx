import * as React from "react";

export interface IHomeProps {}

export function Home(props: IHomeProps) {
  return <div></div>;
}
