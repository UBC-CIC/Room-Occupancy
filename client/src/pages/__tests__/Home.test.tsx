import * as React from "react";
import { render } from "@testing-library/react";
import { Home } from "../Home";
import { BrowserRouter as Router } from "react-router-dom";

describe("<Home />", () => {
  it("matches snapshot", () => {
    const { asFragment } = render(
      <Router>
        <Home />
      </Router>
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
