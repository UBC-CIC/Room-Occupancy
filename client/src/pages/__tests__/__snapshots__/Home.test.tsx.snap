// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <h1
      class="ui center aligned header"
    >
      <div
        class="content"
      >
        Real Time Occupancy Management
      </div>
      <div
        class="sub header"
        color="blue"
      >
        Utilize IoT devices to understand and analyze occupancy in areas
      </div>
    </h1>
    <div
      class="ui basic segment"
    >
      <img
        class="ui big centered image"
        src="test-file-stub"
      />
    </div>
    <a
      href="/analytics"
    >
      <button
        class="ui blue circular button"
      >
        Check Analytics
      </button>
    </a>
  </div>
</DocumentFragment>
`;
